### DISCUSSION TITLE
Mining

### DISCUSSION THREAD

**Discussion by Arkad00** [Chia-Network/chia-blockchain#5113] [Q&A]
Friend please help with the calculation of the Formula Condition (example) . Chia price 1000$ what equipment is needed to get 500$ per month . I understand that this will not be accurate, please help, until the equipment has become more expensive. Thanks, buddy.

**Comment by lvcivs** [comment]
I think you are looking for the https://chiacalculator.com/ ?

**Comment by Arkad00** [comment]
Re: [Chia-Network/chia-blockchain] Mining (#5113)



Здравствуйте, lvcivs.
thank you friend for your help
Вы писали 15 мая 2021 г., 20:38:51:





I think you are looking for the https://chiacalculator.com/ ?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.





--
С уважением,
 Fins                         ***@***.***
